docs/www/report.html

```diff
@@ -154,7 +154,7 @@ <h2>
 </pre>
    <div class="pull-right">
     <button class="btn btn-default" data-clipboard-target="#report-text">
-     <i alt="Copy to clipboard" class="glyphicon glyphicon-copy">
+     <i class="glyphicon glyphicon-copy" title="Copy to clipboard">
```

Comment:
Is title better for 508 compliance than alt? I would have thought the opposite? Why is it better?

Reply:
There's a lot of funky history with the `<i>` element, but it is not an image as you might think. Technically, it's used for alternate text styling (like italics). It is odd that it's used to render icons, but it is what it is. That said, I don't think the alt attribute would work here on a screen reader. The title attribute does get read with screen readers and also provides that text as a tooltip as an added bonus.

Reply:
That's interesting.